projects/Mallard/src/components/onboarding/onboarding-card.tsx

```diff
@@ -68,63 +69,83 @@ const OnboardingCard = ({
     style?: StyleProp<ViewStyle>
     appearance: CardAppearance
     size?: 'big' | 'small'
-}) => (
-    <View style={style}>
-        <View style={[styles.square, appearances[appearance].background]}>
-            <View style={{ flexGrow: 1 }}>
-                <TitlepieceText
-                    accessibilityRole="header"
-                    style={[
-                        getFont('titlepiece', size === 'big' ? 2.5 : 2.0),
-                        { marginBottom: size === 'big' ? 16 : 8 },
-                        appearances[appearance].text,
-                    ]}
-                >
-                    {title}
-                </TitlepieceText>
-                {subtitle && (
+}) => {
+    const max = minScreenSize() * 0.8
+    return (
+        <View
+            style={[
+                appearances[appearance].background,
+                {
```

Comment:
Should this be pulled out into a Stylesheet?

Reply:
Yes it should 👌